crawl/dependency.py

```diff
@@ -133,6 +133,16 @@ def __repr__(self):
 
 
 class ThirdPartyDependency(AbstractDependency):
+    @classmethod
+    def from_coordinates(cls, bazel_label_name, coord_str):
+        parts = coord_str.split(':')
+        if len(parts) == 3:
+            return cls(bazel_label_name, parts[0], parts[1], parts[2])
+        if len(parts) == 4:
+            return cls(bazel_label_name, parts[0], parts[1], parts[3], scope=parts[2])
+        if len(parts) == 5:
+            return cls(bazel_label_name, parts[0], parts[1], parts[4], scope=parts[2], classifier=parts[3])
+        raise Exception('Illegal maven coordinate: %s' % coord_str)
```

Comment:
Is this similar to new_dep_from_maven_art_str below?  Can we remove one of those methods?  Docstring?

Reply:
Also probably good to add a test for this method specifically?

Reply:
Didn't see the new_dep_from_maven_art_str - I will remove the class method